enso/experiment/tiny_net.py

```diff
@@ -0,0 +1,79 @@
+import haiku as hk
+import jax.numpy as jnp
+import jax
+import numpy as np
+from sklearn.preprocessing import LabelBinarizer
+
+class BetterLabelBinarizer():
+
```

Comment:
LabelBinarizer changes behavior when n_classes=2 and kind of arbitrarily assigns one class to be positive and one to be negative.  This class changes that behavior to output a shape (1, 2) one hot label per example at n_classes=2.